templates/docs/utilities/vertical-spacing.md

```diff
@@ -8,6 +8,24 @@ context:
 
 <hr>
 
+A preferable way to control bottom spacing of an element is using Vanilla pre-defined classes: `.u-sv--shallow`, `.u-sv--regular` and `.u-sv--deep`:
```

Comment:
The "preferable" way is still using spacing coming from components, so would be good to reword it a bit.

```suggestion
When desired vertical spacing between sections of the content can't be achieved using the default margins and paddings that are built into components, you can the bottom spacing of an element is using the following spacing utilities: `.u-sv--shallow`, `.u-sv--regular` and `.u-sv--deep`.
```